pytensor/xtensor/shape.py

```diff
@@ -506,3 +506,56 @@ def expand_dims(x, dim=None, create_index_for_new_dim=None, axis=None, **dim_kwa
         x = Transpose(dims=tuple(target_dims))(x)
 
     return x
+
+
+class XBroadcast(XOp):
+    """Broadcast multiple XTensorVariables against each other."""
+
+    __props__ = ("exclude",)
+
+    def __init__(self, exclude=None):
+        self.exclude = exclude
+
+    def make_node(self, inputs):
```

Comment:
Should expect unpacked inputs
```suggestion
    def make_node(self, *inputs):
```